packages/genesys-spark-components/src/components/stable/gux-accordion/gux-accordion-section/gux-accordion-section.scss

```diff
@@ -51,53 +54,37 @@ section.gux-disabled {
   ::slotted([slot='header']) {
     padding: 0;
     margin: 0;
-    font-family: var(
-      --gse-ui-accordion-header-label-defaultText-fontFamily
-    ) !important;
-    font-size: var(
-      --gse-ui-accordion-header-label-defaultText-fontSize
-    ) !important;
-    font-weight: var(
-      --gse-ui-accordion-header-label-defaultText-fontWeight
-    ) !important;
-    line-height: var(
-      --gse-ui-accordion-header-label-defaultText-lineHeight
-    ) !important;
-    color: var(--gse-ui-accordion-header-default-foreground-labelColor);
+    font-family: ui.$gse-ui-accordion-header-label-defaultText-fontFamily !important;
+    font-size: ui.$gse-ui-accordion-header-label-defaultText-fontSize !important;
+    font-weight: ui.$gse-ui-accordion-header-label-defaultText-fontWeight !important;
+    line-height: ui.$gse-ui-accordion-header-label-defaultText-lineHeight !important;
+    color: ui.$gse-ui-accordion-header-default-foreground-labelColor;
   }
 
   ::slotted([slot='subheader']) {
     // TODO COMUI-2370. Missing tokens for subheaders text styling.
     padding: 0;
-    margin-block: var(--gux-spacing-3xs) 0;
+    margin-block: global.$gse-core-spacing-5xs 0;
```

Comment:
This is an example of the type of mistakes this change will catch for us. The `--gux-spacing-3xs` custom property does not exist and was just failing silently